README.rst

```diff
@@ -7,7 +7,7 @@ WIP
 This driver is under development. Changes may happen until we have an official release
 
 .. image:: https://img.shields.io/travis/faunadb/faunadb-python/master.svg?maxAge=21600 :target: https://travis-ci.org/faunadb/faunadb-python
-.. image:: https://img.shields.io/coveralls/faunadb/faunadb-python/master.svg?maxAge=21600 :target: https://coveralls.io/github/faunadb/faunadb-python?branch=master
+.. image:: https://codecov.io/gh/faunadb/faunadb-python/branch/master/graph/badge.svg :target: https://codecov.io/gh/faunadb/faunadb-python
```

Comment:
The image link is good, but for some reason it's a broken image link in the rendered view.